serious/lib/serious/article.rb

```diff
@@ -130,7 +130,14 @@ def chapters_url
   end
 
   def chapter
-    return @chapter ||= Background.get_chapters(chapters_url)
+    $chapter_chache ||= {}
+    if $chapter_chache.key?(chapters_url)
+      @chapter = $chapter_chache[chapters_url]
```

Comment:
warum überhaupt @chapter? 

Das bleibt doch alles lokal in der methode?

Reply:
good point fixed it to keep it similar to the other stuff in there

Reply:
tbf `@chapter` wird vermutlich selten gesetzt sein aber naja egal .___.